README.rst

```diff
@@ -75,8 +75,46 @@ that the tests pass and also the code follows the code style properly. The
 Configuring and Running
 =======================
 
-To be done... integrade is just starting and in the future you can see how to
-configure and run the functional tests in this section.
+Integrade can be configured to test any instance of cloudigrade. The
+**REQUIRED** environment variables are::
+
+    CLOUDIGRADE_BASE_URL # base url without http/https prefix
+    CLOUDIGRADE_TOKEN    # This is the token of the super user created for
+                         # cloudigrade
+
+The **OPTIONAL** environment variables are::
+
+    CLOUDIGRADE_API_VERSION # defaults to 'v1'
+    CLOUDIGRADE_USER        # defaults to 'admin'
+    CLOUDIGRADE_USER_PASS   # at the moment we only use the token to
```

Comment:
Is this already implemented in code? Maybe we can add this when the feature is available.